.bazelrc

```diff
@@ -0,0 +1,23 @@
+# Do not edit this file without a review from @DataDog/agent-build
+
+build --disk_cache=/tmp/bazel
+build --verbose_failures
+
+# Set up bazel HOST platform specific flags
+#
+# These configurations set flags for the OS we are building *on*, not the OS we
+# are building *for*.
+common --enable_platform_specific_config
+build:linux --sandbox_add_mount_pair=/tmp
+build:macos --sandbox_add_mount_pair=/var/tmp
+build:windows --sandbox_add_mount_pair=C:\Temp
+# Point to the mingw location and other settings that point to local stuff
+# TODO: This should eventually go somewhere else (to avoid requiring specific environment setups)
+build:windows --action_env=PATH --define=MINGW_PATH=c:/tools/msys64/mingw64 --define=GCC_VERSION="12.2.0"
```

Comment:
We may prefer to leave this out until we actually include the toolchain bits that these define's refer to.

Reply:
Removed.